resources/settings/settings.xml

```diff
@@ -19,4 +19,62 @@
     <setting propertyKey="@Properties.OpenWeatherLon" title="@Strings.OpenWeatherLonTitle" prompt="4.896856834375711">
         <settingConfig type="alphaNumeric"/>
     </setting>
+    <setting propertyKey="@Properties.DateFormat" title="@Strings.DateFormatTitle">
+        <settingConfig type="list" required="true">
+            <listEntry value="0">dd MM(English)</listEntry>
+            <listEntry value="1">MM/dd</listEntry>
+        </settingConfig>
+    </setting>
+    <setting propertyKey="@Properties.SecondTimeFormat" title="@Strings.SecondTimeFormatTitle">
+        <settingConfig type="list" required="true">
+            <listEntry value="0">-12:00</listEntry>
```

Comment:
Please modify this so that not an index but a direct value is stored, in this case -12, -11, etc. then we can delete the array

Reply:
The reason for doing this here is that numbers such as 3.5 and 4.5 will be considered as strings in value, which makes it impossible for me to compile the code.Therefore, should time zones like 3:30 be deleted?